### DISCUSSION TITLE
Does Strapi automatically sync Collection-Types between Different Database environments ?

### DISCUSSION THREAD

**Discussion by mudassar-islam** [strapi/strapi#7525] [Legacy Topics]
I have a strapi app with a Development and Production environment. 

I was wondering If Strapi will sync all the database tables (excluding data) and things like Permission Roles and such into the other environment.

Also I have noticed that when I save a collection-type, It works perfectly on the server but the Client shows "An Error has Occured" but refreshes back to normal automatically. I wonder if someone else had this happen to them.

**System:**
OS: **Windows 10 64-bit**
NodeJS: **12.18.2**
NPM: **6.14.5**

**Comment by derrickmehaffy** [comment]
Model settings themselves will be created automatically but it will not "clean-up" deleted fields from the database nor will it rename existing ones (see this issue thread: https://github.com/strapi/strapi/issues/1114 ), this is something we will be looking at this year to resolve in our Q3/4 database refresh.

Data, as you said, is entirely excluded, currently the only export/import tool is for the Content-Manager "views" and other various keys in the `core_store` table/collection. I have already mentioned to our backend engineers about adding in export/import options for permissions but no ETA on that
